### ISSUE TITLE
Should not back up private SSH keys by default

### ISSUE THREAD

**Issue by lazerwalker** [lra/mackup#512]
The default behavior of `mackup` currently includes backing up the `~/.ssh` dir, including all SSH private keys. There are **serious** security implications to uploading your SSH private key unencrypted to a third-party server (e.g. Dropbox).

When I ran `mackup backup`, I had no idea that would happen, and certainly would not have expected it from a cursory reading of the README. This is completely unacceptable. I'm incredibly lucky that I did not have Dropbox.app running at the time, or else I would have felt the need to regenerate my private key, causing me a GREAT deal of hassle and making me INCREDIBLY angry at your software.

I'd **strongly** recommend that SSH backup be disabled by default. My ideal solution would probably involve mackup encrypting everything before moving/uploading (e.g. into a TrueCrypt volume), but given the current symlinking behavior I'd assume that would involve a fair bit of work. 

Some users may be okay with the current behavior. That's fine, but it's not a decision to take lightly, and one that many users will decidedly NOT want. Whatever the solution looks like, this absolutely needs to be an opt-in behavior (if it exists at all).

**Comment by vitorgalvao** [comment]
This has been repeated [time](https://github.com/lra/mackup/issues/336) and [time](https://github.com/lra/mackup/issues/109) again. There should be at least one more issue about this. @lra has also stated that syncing his ssh keys is part of the reason he started mackup in the first place, so this is unlikely to change.

Why these issues are still open, I have no idea.

**Comment by lra** [comment]
You are right, I'll try to keep the discussion centralized.
Regarding the matter, I'm sold to the idea that it should not sync it by default, we just need to name the enabling/disabling of apps simple enough so the empty default application list can be easily modified.

Let's follow up at https://github.com/lra/mackup/issues/109

**Comment by aristidesfl** [comment]
Mackup should actually not move any files without confirming with the user first. It should present the list of apps or files which will be backed up and confirm if the user wants to proceed.

**Comment by lra** [comment]
> Mackup should actually not move any files without confirming with the user first.

That's an opinion, check the issues and some people want the opposite: to not have to be bothered.

**Comment by aristidesfl** [comment]
How often do you run mackup backup?

**Comment by lra** [comment]
once per computer + on every new app installed.

**Comment by enricodeleo** [comment]
I found Mackup back in the days during a research about ssh key syncing :)

I've been using it happily since then. With the last computer I configured I found that the .ssh directory wasn't syncing my keys. I suppose the "no party" won at the and, but I still think that there is no reason not to having shared ssh keys as long as you sync personal computers...
 
I already created a custom config (ssh-keys), luckily for us it is extremely trivial to introduce custom configurations. Great job!
